### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ibmpranati**
Please review and merge my code to main branch.

### CODE REVIEW

File: searchorder/src/main/java/com/ibm/searchorder/OrderService.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 @Service
 public class OrderService {
 	@Autowired
-	OrderRepository orderRepository;
+	OrderRepository orderRepository;	//DI
```
Comment (ibmkunalpawar) [inline searchorder/src/main/java/com/ibm/searchorder/OrderService.java:14]: good code

Comment (ibmpranati) [reply searchorder/src/main/java/com/ibm/searchorder/OrderService.java:14]: Thank you :)

Comment (ibmkunalpawar) [reply searchorder/src/main/java/com/ibm/searchorder/OrderService.java:14]: can u check my change?

Comment (ibmpranati) [reply searchorder/src/main/java/com/ibm/searchorder/OrderService.java:14]: You need to send me the review request for it, I am waiting for it.

**Pull request merged**